src/app/shared/column-chooser/column-chooser.component.ts

```diff
@@ -22,6 +23,7 @@ export class ColumnChooserComponent implements OnInit {
   constructor() { }
 
   ngOnInit() {
+    this.language = JSON.parse(localStorage.getItem("language"));
```

Comment:
We have in help.service.ts method getLanguage for return languages from localStorage
So, we can use it for this.